components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -839,11 +866,11 @@ mod tests {
     }
 
     #[bench]
-    fn recorder_sample(b: &mut test::Bencher) {
-        let mut samples = vec![Sample::new(b"c")];
+    fn samples_evaluate(b: &mut test::Bencher) {
```

Comment:
Can you show the bench test result in this PR?

Reply:
Addressed. PTAL.